DependencyInjection/DoctrineExtension.php

```diff
@@ -489,8 +490,15 @@ protected function ormLoad(array $config, ContainerBuilder $container)
 
         $container->setParameter('doctrine.default_entity_manager', $config['default_entity_manager']);
 
-        $options = ['auto_generate_proxy_classes', 'proxy_dir', 'proxy_namespace'];
+        $options = ['auto_generate_proxy_classes', 'enable_lazy_ghost_objects', 'proxy_dir', 'proxy_namespace'];
         foreach ($options as $key) {
+            if ($key === 'enable_lazy_ghost_objects' && $config[$key] && ! method_exists(OrmConfiguration::class, 'setLazyGhostObjectEnabled')) {
```

Comment:
We should also check if VarExporter and Persistence have been installed in the correct versions.

Reply:
this is not needed: the checks are already done in setLazyGhostObjectEnabled

Reply:
Yes, but we would turn a runtime error into a compile-time one.

Reply:
Good call, PR updated.